src/app/modules/group/components/user-list/user-list.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { Group } from '../../http-services/get-group-by-id.service';
+import { GetGroupMembersService, Member } from '../../http-services/get-group-members.service';
+
+interface MemberAddition {
+  memberSinceText: string,
+}
+
+@Component({
+  selector: 'alg-user-list',
+  templateUrl: './user-list.component.html',
+  styleUrls: [ './user-list.component.scss' ]
+})
+export class UserListComponent implements OnChanges {
+
+  @Input() group? : Group;
+  state: 'loading' | 'error' | 'empty' | 'ready' = 'loading';
+
+  members: (Member&MemberAddition)[] = [];
+
+  constructor(private getGroupMembersService: GetGroupMembersService) { }
+
+  ngOnChanges(_changes: SimpleChanges): void {
+    this.reloadData();
+  }
+
+  private reloadData(): void {
+    if (this.group) {
+      this.state = 'loading';
+
+      this.getGroupMembersService.getGroupMembers(this.group.id)
```

Comment:
As you did at the other places, probably you need to keep the subscription to cancel it if it changes to prevent request reordering causing the wrong content to be displayed.

Reply:
fixed